src/pages/embed/[uid].tsx

```diff
@@ -1,11 +1,19 @@
 import dynamic from "next/dynamic";
-import { useRouter } from "next/router";
+import type {
+  GetServerSidePropsContext,
+  InferGetServerSidePropsType,
+} from "next/types";
 
 import React, { useState } from "react";
 import { PusherProvider, useSubscribeToEvent } from "../../utils/pusher";
+import { prisma } from "../../server/db/client";
 
-const useLatestPusherMessage = (userId: string) => {
-  const [latestMessage, setLatestMessage] = useState<string | null>(null);
+type ServerSideProps = InferGetServerSidePropsType<typeof getServerSideProps>;
+
+const useLatestPusherMessage = (pinnedQuestion: string | null) => {
```

Comment:
1. was userId string not used?
2. This should have a better name, maybe `initialPinnedQuestion` or `defaultPinnedQuestion`?

Reply:
1. Correct. 
2. Good point, I'll update that.